ambry-admin/src/main/java/com.github.ambry.admin/GetReplicasForBlobIdHandler.java

```diff
@@ -1,135 +0,0 @@
-/**
- * Copyright 2016 LinkedIn Corp. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- */
-package com.github.ambry.admin;
-
-import com.github.ambry.clustermap.ClusterMap;
-import com.github.ambry.clustermap.PartitionId;
-import com.github.ambry.clustermap.ReplicaId;
-import com.github.ambry.commons.BlobId;
-import com.github.ambry.commons.ByteBufferReadableStreamChannel;
-import com.github.ambry.rest.RestRequest;
-import com.github.ambry.rest.RestResponseChannel;
-import com.github.ambry.rest.RestServiceErrorCode;
-import com.github.ambry.rest.RestServiceException;
-import com.github.ambry.rest.RestUtils;
-import com.github.ambry.router.ReadableStreamChannel;
-import java.io.IOException;
-import java.nio.ByteBuffer;
-import java.util.List;
-import java.util.Map;
-import org.json.JSONException;
-import org.json.JSONObject;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-
-/**
- * Performs the custom {@link AdminBlobStorageService#GET_REPLICAS_FOR_BLOB_ID} operation supported by the admin.
- */
-class GetReplicasForBlobIdHandler {
```

Comment:
It seems get replicas was previously supported as this class existed before. I did not check the old version of the code, so just wonder if this patch is just to rename and bring up this function?